app/views/insured/families/_marriage_fields.html.erb

```diff
@@ -1,24 +1,49 @@
-<div class="special_qle_reasons hidden">
-  <h4> Did you or your spouse have health insurance for at least one day between <%= @qle_date_calc %> and <%= @qle_date %> ? </h4> <br/>
-  <div class = 'moving_radio_btn'>
-    <div class="n-radio-group">
-      <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept1')" class="n-radio-row">
-        <label class="n-radio vertically-aligned-row" for="reason_accept1">
-          <%= radio_button_tag :reason, "no", false, id: 'reason_accept1', class: "n-radio" %>
-          <span class = "n-radio"></span>
-            yes
-        </label>
-      </div>
-      <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept')" class="n-radio-row">
-        <label class="n-radio" for="reason_accept">
-        <%= radio_button_tag :reason, "yes", true, id: 'reason_accept', class: "n-radio" %>
-        <span class="n-radio"></span>
-          no
-        </label>
+<% if @bs4 %>
+  <div class="special_qle_reasons hidden">
+    <label class="weight-n mb-2"><%= l10n("insured.qle_detail.special_reason.marriage", qle_start_date: @qle_date_calc, qle_date: @qle_date) %></label>
+    <div>
+      <div>
+        <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept1')" class="n-radio-row">
+          <label class="weight-n" for="reason_accept1">
+            <%= radio_button_tag :reason, "no", false, id: 'reason_accept1' %>
+            <%= l10n("yes") %>
+          </label>
+        </div>
+        <div tabindex="0" onkeydown="handleButtonKeyDown(event, 'qle_submit_reason')" onfocus="handleSEPRadioButton('reason_accept')" class="n-radio-row">
+          <label class="weight-n" for="reason_accept">
+            <%= radio_button_tag :reason, "yes", true, id: 'reason_accept', class: "n-radio" %>
```

Comment:
radio button text says 'yes' but line below is 'no', same on ln 8

Reply:
yep I noticed this too. this follows legacy behavior and is expected. Guessing it has to do with how the form values are treated, kind of unintuitive but I think adjusting this is out of scope